data/json/vitrification/terrain_vitrified.json

```diff
@@ -0,0 +1,603 @@
+[
+  {
+    "type": "terrain",
+    "id": "t_quietfarm_north_ground",
```

Comment:
If this isn't converted to black glass(& is thus suitable for environments outside of the special), a more generic ID 
 + move of the IDs to the regular furniture_and_terrain folder would be better.

*Also applies to the three chickenwire fences below this